planner/core/common_plans.go

```diff
@@ -771,9 +771,7 @@ func (e *Execute) rebuildRange(p Plan) error {
 					return errors.New("failed to rebuild range: the length of the range has changed")
 				}
 				for i := range x.IndexValues {
-					for j := range ranges.Ranges[i].LowVal {
-						x.IndexValues[i][j] = ranges.Ranges[i].LowVal[j]
-					}
+					copy(x.IndexValues[i], ranges.Ranges[i].LowVal)
```

Comment:
Unrelated change?

Reply:
It is related with the linter upgrading.

Reply:
It is related with the linter upgrading.